samples/core/blur/main.c

```diff
@@ -1168,22 +1186,26 @@ int main(int argc, char *argv[])
         }
 
         /// Subgroup exchange in dual-pass Gaussian blur
-        if (use_subgroup_exchange_relative)
+        if (use_subgroup_exchange_relative && opencl_version_2_0)
         {
             printf("Dual-pass subgroup relative exchange Gaussian blur\n");
 
             kernel_op[0] = '\0';
+            // cl_khr_subgroup_shuffle_relative requires OpenCL 2.0
+            strcat(kernel_op, " -cl-std=CL2.0 ");
```

Comment:
See comment above: there is no dependency between `cl_khr_subgroup_shuffle_relative` and OpenCL C 2.0.